src/config/ethereum.ts

```diff
@@ -1,7 +1,7 @@
 import { Networks } from "./vega";
 
 const appChainId = process.env.REACT_APP_CHAIN as EthereumChainId;
-const appEnv = process.env.REACT_APP_ENV as Networks;
+export const appEnv = process.env.REACT_APP_ENV as Networks;
```

Comment:
Perhaps this pattern is unnecessary but later on in this file we export `appChainId` as `APP_CHAIN_ID`. Might be good to be consistent and export `appEnv` as `APP_ENV` so we follow the 'best practice' of upper camel case for constants

Reply:
Or just define them immediately as upper camel case